def test_bad_calls(self):
    _expect_exception((lambda : client.insert('Keyspace1', 'key1', ColumnPath('Standard1', 'x', 'y'), 'value', 0, ConsistencyLevel.ONE)), InvalidRequestException)
    _expect_exception((lambda : client.get('Keyspace1', 'key1', ColumnPath('Standard1'), ConsistencyLevel.ONE)), InvalidRequestException)
    _expect_exception((lambda : client.get('Keyspace1', 'key1', ColumnPath('Standard1', 'x', 'y'), ConsistencyLevel.ONE)), InvalidRequestException)
    _expect_exception((lambda : client.get('Keyspace1', 'key1', ColumnPath('Super1'), ConsistencyLevel.ONE)), InvalidRequestException)
    _expect_exception((lambda : client.get_key_range('Keyspace1', 'S', '', '', 1000)), InvalidRequestException)
